grafana-plugin/pkg/plugin/settings.go

```diff
@@ -92,11 +101,10 @@ func (a *App) OnCallSettingsFromContext(ctx context.Context) (*OnCallPluginSetti
 
 	if settings.License == "" {
 		cloudRe := regexp.MustCompile(CLOUD_VERSION_PATTERN)
-		cloudDevRe := regexp.MustCompile(CLOUD_DEV_VERSION_PATTERN)
 		ossRe := regexp.MustCompile(OSS_VERSION_PATTERN)
-		if version == DEV_PATTERN || ossRe.MatchString(version) {
+		if ossRe.MatchString(pluginContext.PluginVersion) {
```

Comment:
Should we still use the `version` value here? (added to support previous Grafana versions where version wasn't available in context)

Reply:
Good catch! probably why the tests are failing

Reply:
Fixed by @brojd